### DISCUSSION TITLE
Adding HPO terms via Bulk Edit Metadata Option

### DISCUSSION THREAD

**Discussion by anjalijain22** [broadinstitute/seqr#4451] [Q&A]
Hello,

I'm using the "Bulk Edit Individual Metadata" option to add HPO terms for all individuals in my project. I'm able to add the HPO terms such as HP:0001061 through a csv file I prepared but I am unable to add other details for any HPO term such as Age of Onset, Laterality, Pace of progression, comments etc. through the bulk edit option. Is there a specific format that seqr requires the HPO terms to be listed in the csv file, for it to be able to read these specifications and add it to the project page? Currently, I'm only able to add in details from the project page directly by manually filling out each detail for each HPO term:

<img width="700" alt="Screenshot 2024-10-25 at 12 06 08 PM" src="https://github.com/user-attachments/assets/d3b90461-db82-4f92-8b65-8578f75371fd">

It would be amazing if there is a way for me add in additional information for HPO terms through the Bulk edit metadata option.

Thank you in advance for your help,
Anjali

**Comment by hanars** [comment]
seqr does not support bulk updating any of the modifiers for HPO terms (such as Age of Onset and Laterality), only the main terms themselves
